src/compile-middleware.ts

```diff
@@ -115,17 +115,28 @@ export function babelCompile(
         transformModules: forceCompile || !capabilities.has('modules'),
       };
       const optionsLeader = JSON.stringify(options);
-      const cached = babelCompileCache.get(optionsLeader + body);
+      const cached = babelCompileCache.get(optionsLeader + request.path + body);
```

Comment:
Can it be *just* `request.path` now? Add a comment?

Reply:
I'm really wondering why options got in there in the first place, since options should be unique for any running polyserve process. I tried to remove it but got a couple of weird failures. I added a TODO and filed https://github.com/Polymer/polyserve/issues/248 to track.